core/src/main/kotlin/org/evomaster/core/docs/ConfigToMarkdown.kt

```diff
@@ -180,8 +190,30 @@ object ConfigToMarkdown {
             buffer.append(" *Constraints*: `${description.constraints}`.")
         }
         if(description.enumValues.isNotBlank()){
-            buffer.append(" *Valid values*: `${description.enumValues}`.")
+            var experimentalValues = ""
+            var validValues = ""
+            var findVariable = false
+            for (element in defaultInstance.listAnnotationsExperimental) {
+                if (element.variableName == opt.name) {
+
+                    experimentalValues = element.experimentalValues
+                    validValues = element.validValues
+                    findVariable = true
+                }
+            }
+            if (findVariable==true)
+            {
+                buffer.append(" *Valid values*: `${validValues}`, *Experimental Values*: `${experimentalValues}` ")
+            }
+            else
+            {
+
+                buffer.append(" *Valid values*: `${description.enumValues}`, *Experimental Values*: `This is under work` ")
+            }
```

Comment:
if there is no experimental value, then the `*Experimental values*` label should not be printed (also notice lower case for `v`)